can_tools/scrapers/official/federal/CDC/cdc_vaccines.py

```diff
@@ -0,0 +1,89 @@
+import random
+
+import pandas as pd
+import requests
+import us
+
+from can_tools.scrapers import variables
+from can_tools.scrapers.base import ALL_STATES_PLUS_DC, CMU
+from can_tools.scrapers.official.base import FederalDashboard
+
+
+def _lookup(location):
+    if location == "US":
+        return 0
+    return int(us.states.lookup(location).fips)
+
+
+class CDCStateVaccine(FederalDashboard):
+    has_location = True
+    location_type = "state"
+    source = "https://covid.cdc.gov/covid-data-tracker/#vaccinations"
+    source_name = "Centers for Disease Control and Prevention"
+    provider = "cdc"
+
+    variables = {
+        "Doses_Distributed": variables.TOTAL_VACCINE_DISTRIBUTED,
+        "Administered_Dose1_Recip": variables.INITIATING_VACCINATIONS_ALL,
+        "Series_Complete_Yes": variables.FULLY_VACCINATED_ALL,
+        "Doses_Administered": variables.TOTAL_DOSES_ADMINISTERED_ALL,
+    }
+
+    def fetch(self, test=False):
+        fetcher_url = (
+            "https://covid.cdc.gov/covid-data-tracker/COVIDData/"
+            "getAjaxData?id=vaccination_data"
+        )
+        response = requests.get(fetcher_url)
+
+        return response.json()
+
+    def _filter_rows(self, df):
+        state_abbr_list = [x.abbr for x in ALL_STATES_PLUS_DC]
```

Comment:
It looks like CDC has Puerto Rico and Mariana Islands data.  Any idea if there's a reason we filter them out?  It's tempting to pull them in, though a separate PR would be fine if we want to reduce moving pieces.

Reply:
will do in new pr